### ISSUE TITLE
Installing 0.6.0 in Atom Windows 7 fails

### ISSUE THREAD

**Issue by jaredmorgs** [asciidoctor/atom-asciidoc-preview#113]
I tried to get a Windows 7 user started with AsciiDoc today and attempted to install a-a-p 0.6.0, but it wouldn't install. 

I could not see the root cause of the failure, but I wonder if it relates to #112 that has crept into the latest release.

I did all the usual things like restarting Atom after the initial install, but it made no real difference.

**Comment by DanielGrigoras** [comment]
I also tried installing just now “asciidoc-preview@0.6.0” in Atom on Windows 7 x64, but it failed with the following error:

npm WARN deprecated CSSselect@0.4.1: the module is now available as 'css-select'
npm WARN deprecated CSSwhat@0.4.7: the module is now available as 'css-what'
npm ERR! git clone C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 C:\Users\GRIGOR~1\AppData\Local\Temp\npm-2324-ce01adb7\git-cache-8a01cd2114b9\b56048584b3cf2cfebeef78b0fff54690e12e684: cygwin warning:
npm ERR! git clone C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 C:\Users\GRIGOR~1\AppData\Local\Temp\npm-2324-ce01adb7\git-cache-8a01cd2114b9\b56048584b3cf2cfebeef78b0fff54690e12e684:   MS-DOS style path detected: C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/.git
npm ERR! git clone C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 C:\Users\GRIGOR~1\AppData\Local\Temp\npm-2324-ce01adb7\git-cache-8a01cd2114b9\b56048584b3cf2cfebeef78b0fff54690e12e684:   Preferred POSIX equivalent is: /cygdrive/c/Users/grigorasd/.atom/.apm/_git-remotes/git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/.git
npm ERR! git clone C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 C:\Users\GRIGOR~1\AppData\Local\Temp\npm-2324-ce01adb7\git-cache-8a01cd2114b9\b56048584b3cf2cfebeef78b0fff54690e12e684:   CYGWIN environment variable option "nodosfilewarning" turns off this warning.
npm ERR! git clone C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 C:\Users\GRIGOR~1\AppData\Local\Temp\npm-2324-ce01adb7\git-cache-8a01cd2114b9\b56048584b3cf2cfebeef78b0fff54690e12e684:   Consult the user's guide for more details about POSIX paths:
npm ERR! git clone C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 C:\Users\GRIGOR~1\AppData\Local\Temp\npm-2324-ce01adb7\git-cache-8a01cd2114b9\b56048584b3cf2cfebeef78b0fff54690e12e684:     http://cygwin.com/cygwin-ug-net/using.html#using-pathnames
npm ERR! git clone C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 C:\Users\GRIGOR~1\AppData\Local\Temp\npm-2324-ce01adb7\git-cache-8a01cd2114b9\b56048584b3cf2cfebeef78b0fff54690e12e684: fatal: failed to open '/cygdrive/c/Users/grigorasd/.atom/.apm/_git-remotes/git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/objects': No such file or directory
npm ERR! git clone C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 C:\Users\GRIGOR~1\AppData\Local\Temp\npm-2324-ce01adb7\git-cache-8a01cd2114b9\b56048584b3cf2cfebeef78b0fff54690e12e684: 
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Users\\grigorasd\\AppData\\Local\\atom\\app-1.2.1\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\grigorasd\\AppData\\Local\\atom\\app-1.2.1\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\grigorasd\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\grigorasd\\.atom\\.apmrc" "install" "C:\\Users\\GRIGOR~1\\AppData\\Local\\Temp\\d-1151018-8780-195i34y\\package.tgz" "--target=0.34.0" "--arch=ia32" "--msvs_version=2013"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code 128

npm ERR! Command failed: cygwin warning:
npm ERR!   MS-DOS style path detected: C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/.git
npm ERR!   Preferred POSIX equivalent is: /cygdrive/c/Users/grigorasd/.atom/.apm/_git-remotes/git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/.git
npm ERR!   CYGWIN environment variable option "nodosfilewarning" turns off this warning.
npm ERR!   Consult the user's guide for more details about POSIX paths:
npm ERR!     http://cygwin.com/cygwin-ug-net/using.html#using-pathnames
npm ERR! fatal: failed to open '/cygdrive/c/Users/grigorasd/.atom/.apm/_git-remotes/git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/C:\Users\grigorasd\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/objects': No such file or directory
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

**Comment by js402882** [comment]
Same here. I also tried to install `asciidoc-preview@0.6.0` in Atom 1.4.0 on Windows 7 x64 but it failed and gave me this error
```
npm WARN deprecated CSSselect@0.4.1: the module is now available as 'css-select'
npm WARN deprecated CSSwhat@0.4.7: the module is now available as 'css-what'
npm ERR! git clone C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 <tmp_dir>\npm-5968-1025f2c2\git-cache-f705b92e6004\b56048584b3cf2cfebeef78b0fff54690e12e684: Cloning into '<tmp_dir>\npm-5968-1025f2c2\git-cache-f705b92e6004\b56048584b3cf2cfebeef78b0fff54690e12e684'...
npm ERR! git clone C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 <tmp_dir>\npm-5968-1025f2c2\git-cache-f705b92e6004\b56048584b3cf2cfebeef78b0fff54690e12e684: fatal: '/c/Users/<user>/.atom/.apm/_git-remotes/git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3' does not appear to be a git repository
npm ERR! git clone C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 <tmp_dir>\npm-5968-1025f2c2\git-cache-f705b92e6004\b56048584b3cf2cfebeef78b0fff54690e12e684: fatal: Could not read from remote repository.
npm ERR! git clone C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 <tmp_dir>\npm-5968-1025f2c2\git-cache-f705b92e6004\b56048584b3cf2cfebeef78b0fff54690e12e684: 
npm ERR! git clone C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 <tmp_dir>\npm-5968-1025f2c2\git-cache-f705b92e6004\b56048584b3cf2cfebeef78b0fff54690e12e684: Please make sure you have the correct access rights
npm ERR! git clone C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 <tmp_dir>\npm-5968-1025f2c2\git-cache-f705b92e6004\b56048584b3cf2cfebeef78b0fff54690e12e684: and the repository exists.
npm ERR! git clone C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3 <tmp_dir>\npm-5968-1025f2c2\git-cache-f705b92e6004\b56048584b3cf2cfebeef78b0fff54690e12e684: 
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Users\\<user>\\AppData\\Local\\atom\\app-1.4.0\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\<user>\\AppData\\Local\\atom\\app-1.4.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\<user>\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\<user>\\.atom\\.apmrc" "install" "C:\\IBM\\tmp\\d-116026-1360-1fzc2cs\\package.tgz" "--target=0.34.5" "--arch=ia32" "--msvs_version=2010"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code 128

npm ERR! Command failed: Cloning into '<tmp_dir>\npm-5968-1025f2c2\git-cache-f705b92e6004\b56048584b3cf2cfebeef78b0fff54690e12e684'...
npm ERR! fatal: '/c/Users/<user>/.atom/.apm/_git-remotes/git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3' does not appear to be a git repository
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```

I need to mention that I have cygwin installed and it seems that Atom somehow (???) detects that and mixes "cygwin" path with ordinary windows path as you can tell from line:

```
npm ERR! fatal: '/c/Users/<user>/.atom/.apm/_git-remotes/git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3/C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3' does not appear to be a git repository
```
If I check the path
```
C:\Users\<user>\.atom\.apm\_git-remotes\git-github-com-anthonny-opal-npm-wrapper-git-1727a7b3
```
there is what I suppose is bare repo of the `opal-npm-wrapper-git` module. 

Anybody has any idea?

**Comment by ldez** [comment]
Can you retry with the 1.0.0 ?

**Comment by DanielGrigoras** [comment]
I just updated and the preview worked. However, with another asciidoc document, which probably has some sort of a syntax error, at least if I am to trust AsciidocFX, that I could not identify, all I got in the right Preview pane was the message "Loading AsciiDoc...", which seemed to be willing to go on forever.

**Comment by ldez** [comment]
@DanielGrigoras can you open an another issue with a sample to reproduce ?

**Comment by ldez** [comment]
fixed in 1.0.0.
